README.md

```diff
@@ -19,12 +19,12 @@ Examples on how to use PsychroLib in all the supported languages are described i
 
 - Python: from the command prompt with `pip install psychrolib`.
 
+- C#, F#, and Visual Basic: from the [NuGet package](https://www.nuget.org/packages/PsychroLib/) manager or clone the repository, and bundle according to your requirements.
```

Comment:
Since the language the .NET port is written only in C#, it may be clearer to word it similar below:
```
- C#, F#, and Visual Basic: from the [NuGet package](https://www.nuget.org/packages/PsychroLib/) manager.
- C#: clone the repository, and bundle according to your requirements.
```